script.js

```diff
@@ -22,3 +22,223 @@ navLinksEl.forEach((item) => {
+    FeaturedImage: 'images/Snapshoot Portfolio (1).png',
+  },
+
+  {
+    Name: 'Facebook 360',
+    Description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    FeaturedImage: 'images/Snapshoot Portfolio.png',
+  },
+
+  {
+    Name: 'Uber Navigation',
+    Description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    FeaturedImage: 'images/Snapshoot Portfolio (3).png',
+  },
+];
+
+const sectionHide = document.querySelector('.container');
+const btnEl1 = document.querySelector('.btn-one');
+const btnEl2 = document.querySelector('.btn-two');
+const btnEl3 = document.querySelector('.btn-three');
+const btnEl4 = document.querySelector('.btn-four');
+const body = document.querySelector('body');
+
+btnEl1.addEventListener('click', () => {
+  sectionHide.classList.toggle('hide-container');
+  body.classList.toggle('background-img');
+  const Popsection = document.createElement('section');
+  Popsection.className = 'Popup-Section';
+  Popsection.innerHTML = `
+      <div class="icon-holder">
+      <h2>${workCards[0].Name} </h2>
+      <a onclick="showMenu(close)"><i class="fa-solid fa-xmark close-cards"></i></a>
```

Comment:
- [ ] Kindly try to stick the Figma Design at the color of the text here and with that this will make the popup attractive 😊
- [ ]  Also😃try to increase the size of the text here  in the desktop version to be like the Figma Design😊
- [ ] Also kindly try to use the correct font weight of the size here to be like the Figma Design💯


> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|    ![image](https://user-images.githubusercontent.com/83360636/216102135-62bdf948-a0a7-458b-9441-10876146dcdc.png)| ![image](https://user-images.githubusercontent.com/83360636/216102066-43cabafc-3dba-4e28-8b3f-8b2821e91656.png)|